VitalsSimplification/BpmChecker.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace VitalsSimplification
+{
+    class BpmChecker
+    {
+       internal static string vitalName = "BeatPerMinute"; 
```

Comment:
Nicely modularized 👍 
The ...checker classes are very similar. Try making one class out of it, accepting the vitalName and limits in the constructor. Then, we can simply instantiate different objects for different vitals.